backend/estuary/estuary.go

```diff
@@ -436,12 +436,16 @@ func (f *Fs) List(ctx context.Context, dir string) (entries fs.DirEntries, err e
 			if err != nil {
 				return nil, err
 			}
-			o, err := f.newObjectWithInfo(ctx, path.Join(dir, item.Name), parseTimeString(pin.Meta["modTime"].(string)), &content{ID: item.ContentID, Cid: item.Cid, Size: item.Size})
+			modTime := pin.Meta["modTime"]
+			if modTime == nil {
+				// Only object created by rclone will have modTime
```

Comment:
Let's remove this comment actually because this can change in other codebases that we don't have control over.

Reply:
changed it to use current time